README.MD

```diff
@@ -1 +1,110 @@
-Implement Superfarmer board game logic in Java.
\ No newline at end of file
+ Implement Super farmer board game logic in Java.
```

Comment:
it would be worth mentioning that this is from the description of the original game, or put somewhere a reference to the copyright, or a wikipedia link or the link to the game's creator.